tests/tests/rrfs_conus13km_radar_tten_warm

```diff
@@ -0,0 +1,99 @@
+###############################################################################
+#
+#  HRRR physics on 13km domain, with radar-derived MP temperature tendencies for 1 hour
+#
+###############################################################################
+
+export TEST_DESCR="HRRR physics on 13km domain, with radar-derived MP temperature tendencies for 1 hour"
+
+export CNTL_DIR=rrfs_conus13km_radar_tten_warm
+
+export LIST_FILES="sfcf000.nc \
+                   sfcf001.nc \
+                   sfcf002.nc \
+                   atmf000.nc \
+                   atmf001.nc \
+                   atmf002.nc"
+
+export_fv3
+
+export FH_DFI_RADAR='fh_dfi_radar = 0.0, 0.25, 0.50, 0.75, 1.0'
```

Comment:
If `FH_DFI_RADAR` was set in the same way as the other namelist parameters, this would just be
```
export FH_DFI_RADAR='0.0, 0.25, 0.50, 0.75, 1.0'
```

Reply:
Not possible in this case.

Reply:
Can we set `export FH_DFI_RADAR='-2e10, -2e10, -2e10,-2e10,-2e10'` in defaultvar.sh and then we can set the variables as Dom suggested?

Reply:
You only need to specify one since the rest already have that value.